common/ts/publish-task.ts

```diff
@@ -36,6 +36,9 @@ export default async function publishTask(endpointType: EndpointType) {
   }
 
   tl.debug(`Number of analyses in this build: ${taskReports.length}`);
+  if (!taskReports.length) {
```

Comment:
Hi, 

Thank you for this proposition. I suggest to slightly modify it, and add this check / warning before the debug line (38), and only display this line in case of any report found.

Reply:
Done!